tooling/greybox_fuzzer/src/coverage.rs

```diff
@@ -532,6 +516,14 @@ impl AccumulatedFuzzerCoverage {
     ) {
         // Insert all ACIR states and replace testcase association
         for acir_bool_state in new_coverage.acir_bool_coverage.iter() {
+            // If the witness is non-boolean according the merged non-bool witness list, skip it
+            if self
+                .non_bool_witness_list
+                .witness
+                .contains(&Witness::new(acir_bool_state.witness_id))
```

Comment:
Extracting this argument to a local variable reduces the number of lines a bit.

Reply:
Done